Package.swift

```diff
@@ -90,6 +90,9 @@ private extension Package {
         ]
 
         let sdksToIncludeInTargets = try! FileManager.default.contentsOfDirectory(atPath: localReleaseSwiftSDKDir.path)
+            .filter { target in
+                !target.hasPrefix(".DS_Store")
```

Comment:
Should this maybe be a helper method?  Also, more generally, we would likely want to exclude any dot file, not just `.DS_Store`

Reply:
Although not need but fair to include all files with `.` prefix.

Reply:
in a Package.swift file another helper methods seem overkill.